packages/plugins/experimental/plugin-transcription/src/types/schema.ts

```diff
@@ -18,7 +18,7 @@ export const TranscriptSchema = S.Struct({
    * Queue containing TranscriptBlock objects.
    */
   // TODO(wittjosiah): Should be a ref to a queue.
-  queue: S.optional(S.String),
+  queue: Ref(Expando),
```

Comment:
Could it cause any problems, if ECHO tries to load it?

Reply:
yes, if you try to call `.target` on this ref, it throws. However this is the preferred way to reference queues in ECHO right now as it best represents the structure of the data and the expectation is that the api will evolve to handle this case better. We've used this case in a number of other places already